new.php

```diff
@@ -47,6 +47,8 @@
         p2 int NOT NULL,
         winner int,
         player VARCHAR(20), 
+        pairing_id INT NOT NULL,
```

Comment:
My bad, the pairing ID is already there in the ID auto increment primary key tag. The winner should be a boolean, which is true if player1 won. It should be able to be null. iscomplete looks good though